ansible_runner/runner_config.py

```diff
@@ -188,18 +185,16 @@ def prepare(self):
             self.command = self.wrap_args_with_ssh_agent(self.command, self.ssh_key_path)
 
         # Use local callback directory
-        callback_dir = self.env.get('AWX_LIB_DIRECTORY', os.getenv('AWX_LIB_DIRECTORY'))
-        if callback_dir is None:
-            callback_dir = os.path.join(os.path.split(os.path.abspath(__file__))[0],
-                                        "callbacks")
-        python_path = self.env.get('PYTHONPATH', os.getenv('PYTHONPATH', ''))
-        if python_path and not python_path.endswith(':'):
-            python_path += ':'
-        self.env['ANSIBLE_CALLBACK_PLUGINS'] = ':'.join(filter(None,(self.env.get('ANSIBLE_CALLBACK_PLUGINS'), callback_dir)))
+        runner_root = os.path.split(os.path.abspath(__file__))[0]
+        collection_dir = os.path.join(runner_root, 'lib')
+
+        # The new method with collection
+        self.env['ANSIBLE_COLLECTIONS_PATH'] = ':'.join(filter(None,(self.env.get('ANSIBLE_COLLECTIONS_PATHS'), collection_dir)))
+
         if 'AD_HOC_COMMAND_ID' in self.env:
-            self.env['ANSIBLE_STDOUT_CALLBACK'] = 'minimal'
+            self.env['ANSIBLE_STDOUT_CALLBACK'] = 'runner.wrapper.minimal'
```

Comment:
Why is it `runner.wrapper.minimal` here and `ansible.runner.minimal` above?

Reply:
A goof.

I had meant to change everything to `runner.wrapper` because being in the `ansible` namespace carries with it a number of connotations which I do not wish to imply here.